### ISSUE TITLE
Clarify if SC 2.4.4 requires paragraph to be a P element in HTML

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2109]
SC 2.4.4 allows for link purpose within a paragraph.   Paragraph is not defined - so generally we fall to the dictionary definition of paragraph which appears to be:

"a distinct section of a piece of writing, usually dealing with a single theme and indicated by a new line, indentation, or numbering."

In that case a link within a span, section, div, p, etc. could still pass as being in a requirement if it meet the above.

Related to issues #1745

**Comment by detlevhfischer** [comment]
In my book the difinition would apply to `div` the same way as it applies to `p`.
I have included an example for both sentence in `p` and in `div` at the bottom of http://3needs.org/en/testing/code/link-issues.html
In Jaws, _ALT + up arrow_ behaves the same in both cases (reading sentence up to the link focused).

**Comment by scottaohara** [comment]
should reference HTML to make it more clear that paragraph is a `p`, but **doesn't have to be** a `p`.

https://html.spec.whatwg.org/multipage/dom.html#paragraphs

**Comment by mraccess77** [comment]
JAWS also supports the control+numpad 5 command to read the current paragraph for divs and other flow content.

**Comment by detlevhfischer** [comment]
> JAWS also supports the control+numpad 5 command to read the current paragraph for divs and other flow content.

Strange - that was the first command I tried but that didn't work here (just red the link itself)...
